.changeset/pink-bees-provide.md

```diff
@@ -0,0 +1,8 @@
+---
+'@aws-amplify/deployed-backend-client': minor
```

Comment:
Why only this one is `minor`?

Reply:
Because of API changes to the `BackendOutputClientErrorType` enum this package becomes `minor` instead of `patch`